stdlib/public/core/IntegerParsing.swift

```diff
@@ -102,7 +102,7 @@ extension FixedWidthInteger {
   ///
   /// The string passed as `text` may begin with a plus or minus sign character
   /// (`+` or `-`), followed by one or more numeric digits (`0-9`) or letters
-  /// (`a-z` or `A-Z`). The string is case insensitive.
+  /// (`a-z` or `A-Z`). The string is case-insensitive.
```

Comment:
@natecook1000 You're right! My personal style is to hyphenate most compound adjectives (with the most common exception arising when the first word is an adverb that ends in "-ly"). However, _Chicago Manual_ rules (which, I believe, is or was at one point Apple's house style) calls for the spelling "case insensitive" when it follows the noun.

Also an astute point about what exactly is case-insensitive. I'd tweak it a little and write: "Parsing of the string is case insensitive."